### PR TITLE
System manager distributor

### PR DESCRIPTION
**Pull request by 4demos**
# Summary

A concise description of the changes being introduced. If you are submitting a PR for a
new quickstart please review the pre merge checklist section to validate this PR is ready for review.

## Pre merge checklist

- [ ] Did you check you NRQL syntax? - Does it work?
- [ ] Did you check your dashboard image quality? -  Do they look good?
- [ ] Did you check that your alerts actually work?
- [ ] Did you include an InstallPlan and Documentation reference?
- [ ] Did you check your descriptive content for voice, tone, spelling and grammar errors?
- [ ] Did you attach images of your dashboards to the PR so we can see them working?

### Screenshots

Attach images of any visual changes, such as a dashboard here.

### CODE REVIEW

File: quickstarts/aws/aws-smd/config.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+slug: aws-system-manager-distributor
+description: |
+  ## Install New Relic Infrastructure Agent on a diverse fleet of AWS EC2 instances all at once!
+  ## Get started!
+  Check out the [documentation](https://docs.newrelic.com/docs/integrations/amazon-integrations/aws-integrations-list/aws-system-manager-distributor) to learn
```
Comment (josephgregoryii) [inline quickstarts/aws/aws-smd/config.yml:5]: Hey @4demos - looks like this link doesn't exist or might be incorrect. Would you mind double checking the link?

File: quickstarts/aws/aws-smd/config.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+level: New Relic
+authors:
+  - New Relic
+title: Amazon System Manager Distributor
+documentation:
+  - name: System Manager Distributor integration docs
+    description: |
+      Faster and robust installation of New Relic Infrastructure Agent with Amazon System Manager Distributor for new and existing New Relic Customers
+    url: >-
+      https://docs.newrelic.com/docs/integrations/amazon-integrations/aws-integrations-list/aws-system-manager-distributor
```
Comment (josephgregoryii) [inline quickstarts/aws/aws-smd/config.yml:19]: Same url here as well

**Review by josephgregoryii — COMMENTED** [review]
Hey @4demos - Couple comments below. 

Also we wanted to call out that since this quickstart does not have any dashboards, alerts, or install plans, this quickstart page will only lead them to the docs page, where in this case would be a broken link.

File: quickstarts/aws/aws-smd/config.yml (hunk 2 above)
Comment (4demos) [reply quickstarts/aws/aws-smd/config.yml]: URL has been updated

File: quickstarts/aws/aws-smd/config.yml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+slug: aws-system-manager-distributor
+description: |
+  ## Install New Relic Infrastructure Agent on a diverse fleet of AWS EC2 instances all at once!
+  ## Get started!
+  Check out the [documentation](https://docs.newrelic.com/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-sys-dist/) to learn 
+  more about New Relic monitoring and integration with AWS System Manager Distributor.
+summary: |
+  Use New-Relic-infrastructure-monitoring-agent tile to install the agent and monitor your EC2 instances with New Relic
```
Comment (aswanson-nr) [inline quickstarts/aws/aws-smd/config.yml:8]: The wording here is a little suspect, I don't think user's will understand what a "tile" is, I'm not quite sure myself. If it's referring to the "tile" in the IO catalogue page view, then we definitely don't want to refer to it that way. I'd like to get this updated before we publish it.

File: quickstarts/aws/aws-smd/config.yml [hunk 4]
```diff
@@ -0,0 +1,23 @@
+slug: aws-system-manager-distributor
+description: |
+  ## Install New Relic Infrastructure Agent on a diverse fleet of AWS EC2 instances all at once!
+  ## Get started!
+  Check out the [documentation](https://docs.newrelic.com/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-sys-dist/) to learn 
+  more about New Relic monitoring and integration with AWS System Manager Distributor.
```
Comment (aswanson-nr) [inline quickstarts/aws/aws-smd/config.yml:6]: I think we could get a little more information in here before tossing the user out to the docs site. We want users to understand the value provided by this quickstart before they need to open links to other pages.

**Review by aswanson-nr — CHANGES_REQUESTED** [review]
I found a few issues with the description and summary, mostly content related. Probably just need to reword them a little. 
I also noticed that this quickstart does not include any dashboards or alerts, meaning that all this will do is link user's out to the link in the `documentation` field. We generally try to include some extra value for users, so that there's a reason to get started using the quickstart. Are there any dashboards we can provide that display the data generated by this integration?

**Comment by seemantk** [comment]
Let's park this PR until @4demos and I sync up on this effort

**Comment by aswanson-nr** [comment]
Moving this to draft until it is ready for review

**Pull request closed without merging**